src/Controls/src/Core/Handlers/Items/iOS/ItemsViewLayout.cs

```diff
@@ -113,6 +113,8 @@ internal virtual bool UpdateConstraints(CGSize size)
 
 			ClearCellSizeCache();
 
+			EstimatedItemSize = CGSize.Empty;
```

Comment:
Do we use empty specifically to know we need to estimate the size? 

Just wanting to make sure we shouldn't be using `UICollectionViewFlowLayout.AutomaticSize` which has a specific meaning (and is not equivalent to Empty).  I am guessing we want Empty here since we do the manually measurement on CV1 still, but wanted to be sure.

Reply:
Looks like no, .
There is a call to `ConstrainTo()` method which is implemented in the `ItemsViewLayout` subclasses. Their implementation calls `DetermineCellSize()` which have some checks for `EstimatedItemSize == CGSize.Empty`. There is a comment saying about <iOS 10 fix (237 line in this file), Using `UICollectionViewFlowLayout.AutomaticSize` can lead to crashes and would require further fixes.